### DISCUSSION TITLE
how to use blur and rounded corners?

### DISCUSSION THREAD

**Discussion by prsh11** [yshui/picom#751] [Q&A]
How do I use rounded corners and blur. I don't have any effect using the default config.

**Comment by noisyBrain** [comment]
> How do I use rounded corners and blur. I don't have any effect using the default config.

Hi man, could you solved it? I'm having the same issues!

**Comment by aspiiire** [comment]
You just have to add `corner-radius = 10;` to your config file, also don't forget to run picom under --experimental-backend

Edit: I have just read that you want to also set blur... so to install blur you have to define it like this

```
blur: {
   method = "dual_kawase";
   strength = 3;
   background = false;
   background-frame = false;
   background-fixed = false;
}
```

You also have to set glx as backed, that as far I know is preinstalled, i'm not an expert, so I cannot give your more info, you do it like this
```
backend = "glx";
```

The most important thing is that you set --experimental-backends when calling picom, so probably after exec picom in your i3 config file

**Comment by ohxxm** [comment]
Doesn't work
